src/Stratis.Features.FederatedPeg/Wallet/FederationWalletManager.cs

```diff
@@ -570,7 +576,7 @@ private void AddTransactionToWallet(Transaction transaction, TxOut utxo, int? bl
             }
             else
             {
-                this.logger.LogTrace("Transaction ID '{0}' found, updating.", transactionHash);
+                this.logger.LogTrace("Transaction ID '{0}' found, updating BlockHeight={2}, BlockHash={3}.", transactionHash, index, blockHeight, block?.GetHash());
```

Comment:
Here there are 4 items specified to be inserted into the string but only 3 is allowed?